docking/rigorous_work.py

```diff
@@ -424,12 +445,20 @@ def main():
     parser.add_argument(
         "--no_outfiles", action="store_true", help="write no output pdb/sdf files"
     )
+    parser.add_argument(
+        "--deletion_steps", type=int, default=501, help="how many steps to delete over"
+    )
+    parser.add_argument(
+        "--num_deletions", type=int, default=10, help="number of deletions to do"
+    )
```

Comment:
Suggest to swap order of these to match the Python signature (although not technically required!)